loadtest/control/simplecontroller/actions.go

```diff
@@ -149,6 +150,38 @@ func (c *SimpleController) searchUsers() control.UserStatus {
 	return c.newInfoStatus(fmt.Sprintf("found %d users", len(users)))
 }
 
+func (c *SimpleController) updateProfile() control.UserStatus {
+	userId := c.user.Store().Id()
+	userName := fmt.Sprintf("testuserNew%d", c.id)
+	nickName := fmt.Sprintf("testNickName%d", c.id)
+	firstName := fmt.Sprintf("firstName%d", c.id)
+	lastName := fmt.Sprintf("lastName%d", c.id)
+	err := c.user.PatchUser(userId, &model.UserPatch{
+		Username:  &userName,
+		Nickname:  &nickName,
+		FirstName: &firstName,
+		LastName:  &lastName,
+	})
+	if err != nil {
+		return c.newErrorStatus(err)
+	}
+	return c.newInfoStatus("user patched")
+}
+
+func (c *SimpleController) updateProfileImage() control.UserStatus {
+	// TODO: take this from the config later.
```

Comment:
Yes, also it might make sense to keep a shared _in memory_ cache for this kind of assets in the future cause we don't want to end up reading the same files 10000 times per minute if we can avoid it :)

Reply:
yes of course. I meant the file path needs to come from the config.